src/components/balance/modals/ModalTransferAmount.vue

```diff
@@ -12,7 +12,7 @@
         class="tw-inline-block tw-bg-white dark:tw-bg-darkGray-900 tw-rounded-lg tw-px-4 sm:tw-px-8 tw-py-10 tw-shadow-xl tw-transform tw-transition-all tw-mx-2 tw-my-2 tw-align-middle tw-max-w-lg tw-w-full"
       >
         <div>
-          <q-banner v-if="isShidenChain" dense rounded class="bg-orange text-white tw-mb-4 q-pa-xs" style="">
+          <q-banner v-if="!isExtrinsicSupported" dense rounded class="bg-orange text-white tw-mb-4 q-pa-xs" style="">
```

Comment:
the name `isExtrinsicSupported` is too broad. Please use something like `isCustomSigBlocked` and make a note that this is just temporary in a comment.